scripts/packageBraveAdsResourcesComponent.js

```diff
@@ -103,18 +104,17 @@
      const crxFile = path.join(crxOutputDir, `user-model-installer-${locale}.crx`)
      const privateKeyFile = path.join(keyDir, `user-model-installer-${locale}.pem`)
      stageFiles(locale, version, stagingDir)
-     util.generateCRXFile(binary, crxFile, privateKeyFile, stagingDir)
+     util.generateCRXFile(binary, crxFile, privateKeyFile, publisherProofKey,
+                          stagingDir)
      console.log(`Generated ${crxFile} with version number ${version}`)
    })
  }
 
  util.installErrorHandlers()
 
- commander
-   .option('-b, --binary <binary>', 'Path to the Chromium based executable to use to generate the CRX file')
-   .option('-d, --keys-directory <dir>', 'directory containing private keys for signing crx files')
-   .option('-e, --endpoint <endpoint>', 'DynamoDB endpoint to connect to', '')// If setup locally, use http://localhost:8000
-   .option('-r, --region <region>', 'The AWS region to use', 'us-west-2')
+ util.addCommonScriptOptions(
+   commander
+     .option('-d, --keys-directory <dir>', 'directory containing private keys for signing crx files'))
```

Comment:
To avoid adding more copy/paste I've extracted the common options to a new function in `util.js`